.vscode/settings.json

```diff
@@ -1,3 +1,4 @@
 {
-	"typescript.tsdk": "node_modules\\typescript\\lib"
+	"typescript.tsdk": "node_modules\\typescript\\lib",
+	"files.eol": "\n"
```

Comment:
`.editorconfig` should already cover this, and if you have the EditorConfig VSCode Plugin it should automatically apply it.  Why do we need it in `settings.json` as well?

Reply:
hmm thats a good question, it seems to be changing otherwise